disorder-jack/src/Disorder/Jack/Combinators.hs

```diff
@@ -122,6 +123,12 @@ sizedIntegral :: Integral a => Jack a
 sizedIntegral =
   mkJack QC.shrinkIntegral QC.arbitrarySizedIntegral
 
+-- | Generates a realfrac number. The number can be positive or negative and
+--   its maximum absolute value depends on the size parameter.
+sizedFloat :: Savage.RealFrac a => Jack a
+sizedFloat =
+  mkJack QC.shrinkRealFrac QC.arbitrarySizedFractional
```

Comment:
FWIW the shrinking function for `RealFrac` changed about a month ago, and doesn't give many of the shrinks it used to. It pretty much just returns Ints now.

https://github.com/nick8325/quickcheck/commit/ca74b9f0c7ad504e197146bfae78209ebf57de1f